whois-api/src/main/java/net/ripe/db/whois/api/elasticsearch/ElasticFullTextIndex.java

```diff
@@ -128,6 +128,15 @@ private SerialEntry getSerialEntry(final int serial) {
         }
     }
 
+    private Map<Integer, Integer> getMaxSerialIdWithObjectCount() {
+        try {
+            return JdbcRpslObjectOperations.getMaxSerialIdWithObjectCount(jdbcTemplate);
+        } catch (Exception e) {
+            LOGGER.debug("Caught exception reading max serial Id with object count", e);
+            return null;
```

Comment:
Returning `null` will cause a nullpointerexception on the next line after the method call. I suggest logging at ERROR and rethrowing the exception.

Reply:
yes, you are right. It will do null.keyset() -> null pointed